documentation/general/dotnet-run-file.md

```diff
@@ -162,20 +160,24 @@ The generated folders might need to be named differently to avoid clashes with e
 
 The entry-point projects (`Program1` and `Program2` in our example)
 have the shared `.cs` files source-included via `<Content Include="../Shared/**/*.cs" />`.
```

Comment:
I am feeling a little concerned about this still, because it doesn't match what a user would ordinarily do with a normal solution setup. But, I think it's the simplest approach, and I don't have any better ideas. It might turn out to be fine.

Reply:
We can also add multiple options to the convert command, e.g., one that generates IVT, or one that generates just project reference (internals won't be visible but maybe user can fix that up manually).